### DISCUSSION TITLE
Any plans for replication/clustering?

### DISCUSSION THREAD

**Discussion by ShadowJonathan** [geldata/gel#1992] [Q&A] [answered]
As per [this old question](https://spectrum.chat/edgedb/general/high-availability-distribution-patterns~237092df-4525-4bca-b27f-377d27caacc5), are there any plans to introduce this?

**Comment by elprans** [comment, marked as answer]
Hi @ShadowJonathan! The current extent of the plans for HA and scalability is discussed in #1859

**Reply by ShadowJonathan** [reply]
Thanks! Sorry for the clutter

**Reply by elprans** [reply]
No worries!
